stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/MessageText.kt

```diff
@@ -122,6 +123,66 @@ public fun MessageText(
     }
 }
 
+/**
+ * Default text element for quoted messages, with extra styling and padding for the chat bubble.
+ *
+ * @param message Message to show.
+ * @param modifier Modifier for styling.
+ */
+@Composable
+public fun QuotedMessageText(
+    message: Message,
+    modifier: Modifier = Modifier,
+    maxQuoteLength: Int = DefaultQuoteMaxLength,
+) {
+    val context = LocalContext.current
+
+    val quotedMessage = when {
+        message.text.count() > DefaultQuoteMaxLength -> {
+            val text = context.getString(
+                R.string.stream_compose_quoted_message_ellipsize,
```

Comment:
We don't have to do manual work for the ellipsis, we can just tell the `Text()` to have an ellipsize `overflow`

Reply:
This will work if we set the maxLines parameter on `Text()`, but for now there is no length check in compose `Text` by which i constrained the message, at least i couldn't find it.